IdentityCore/src/network/request/MSIDAADBrowserPRTResponseSerializer.m

```diff
@@ -0,0 +1,39 @@
+//
+//  MSIDAADBrowserPRTResponseSerializer.m
+//  IdentityCore iOS
+//
+//  Created by Rohit Narula on 1/13/20.
+//  Copyright © 2020 Microsoft. All rights reserved.
+//
+
+#import "MSIDAADBrowserPRTResponseSerializer.h"
+#import "MSIDAADBrowserPRTResponse.h"
+
+@implementation MSIDAADBrowserPRTResponseSerializer
+
+- (id)responseObjectForResponse:(NSHTTPURLResponse *)httpResponse
+                           data:(NSData *)data
+                        context:(id <MSIDRequestContext>)context
+                          error:(NSError **)error
+{
+    if (!httpResponse || !data)
```

Comment:
Can there be a valid case where we receive a response but no data?
e.g. 50x, 30x. kind of responses?